packages/trader/src/index.html

```diff
@@ -60,6 +60,14 @@
 
     <noscript>Your browser does not support JavaScript!</noscript>
 
+    <!-- Start Segment Script -->
+    <script type="text/javascript">
```

Comment:
should be enough to load this script in core/index.html? won't it be for trader then also since core is the "wrapper"

Reply:
@oskar-binary Mahdi will remove it later once further refactoring is done to remove the things not needed from trader, for now we can keep it. at least that's what he said about me adding the meta referrer tag to both core and trader, is it the same in this case @mahdipourziaei-binary ?

Reply:
yes, thanks, i think thats true, but i saw gtm and other script are also doubled. thats why i followed it

Reply:
shouldn't we add it to the bot in that case also?

Reply:
no, bot is already refactored correctly

Reply:
ok 👍